src/content/reference/firmware.md

```diff
@@ -3260,6 +3260,22 @@ UDP udp;
 udp.leaveMulticast();
 ```
 
+### isValid()
+
+Check if the previusly bind UDP socket is valid.
```

Comment:
it is in the base code and I suppose shuld work even for Core and Electron. no way to test.

going to extend example and description ... which will be much way longer that the change itself :-)